react-native/app/index.tsx

```diff
@@ -73,11 +86,7 @@ function Index() {
       } catch (err) {
         let text = "";
         if (err instanceof ConnectError) {
-          if (err.code === Code.Unimplemented) {
-            text = `Hi, ${statement}.  Streaming is not supported in React Native currently.`;
-          } else {
-            text = `A Connect error has occurred: ${err.code} - ${err.message}`;
-          }
+          text = `A Connect error has occurred: ${err.code} - ${err.message}`;
```

Comment:
A ConnectError's message includes the code:

```suggestion
          text = `A Connect error has occurred: ${err.message}`;
```

Can remove the import for `Code` to fix the linter warning.